### PR TITLE
Sanitize Email & Username

### PR DESCRIPTION
**Pull request by NikTripathi**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Sample Data/Custom/GWorkspace_ReportsAPI_calendar_CL.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     },
     "etag": "\"PNA2ZR3TS-DeL8kB7gQOm1H5qhZ72uyN90kw26wJK68/ySNOqSK23I8S3eSyIwVM_YpPIyU\"",
     "actor": {
-      "email": "user4@company.com",
+      "email": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/GWorkspace_ReportsAPI_calendar_CL.json:12]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/GWorkspace_ReportsAPI_drive_CL.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     },
     "etag": "\"PNA2ZR3TS-DeL8kB7gQOm1H5qhZ72uyN90kw26wJK68/ok8bphxpehwyQMjjAojpnEfI07U\"",
     "actor": {
-      "email": "user2@company.com",
+      "email": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/GWorkspace_ReportsAPI_drive_CL.json:12]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/GWorkspace_ReportsAPI_login_CL.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     },
     "etag": "\"PNA2ZR3TS-DeL8kB7gQOm1H5qhZ72uyN90kw26wJK68/qRTs0K2awPrx072aa47TG1SS1bc\"",
     "actor": {
-      "email": "user7@company.com",
+      "email": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/GWorkspace_ReportsAPI_login_CL.json:12]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/GWorkspace_ReportsAPI_mobile_CL.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
     "etag": "\"PNA2ZR3TS-DeL8kB7gQOm1H5qhZ72uyN90kw26wJK68/7atyZfyMUhruHx8ILVKc3eCprTA\"",
     "actor": {
       "callerType": "USER",
-      "email": "user8@company.com",
+      "email": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/GWorkspace_ReportsAPI_mobile_CL.json:13]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/GWorkspace_ReportsAPI_token_CL.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
     },
     "etag": "\"PNA2ZR3TS-DeL8kB7gQOm1H5qhZ72uyN90kw26wJK68/-z59T0Ak79tR1Ovt5CseyUI2H0w\"",
     "actor": {
-      "email": "user4@company.com",
+      "email": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/GWorkspace_ReportsAPI_token_CL.json:12]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/PaloAltoPrismaCloudAlert_CL.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
         "policy_recommendation": "test",
         "policy_labels": "[]",
         "policy_lastModifiedOn": "1616422497101",
-        "policy_lastModifiedBy": "test@example.com",
+        "policy_lastModifiedBy": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/PaloAltoPrismaCloudAlert_CL.json:10]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/PaloAltoPrismaCloudAudit_CL.json [hunk 7]
```diff
@@ -4,89 +4,89 @@
         "user": "00000000-0000-0000-0000-000000000000",
         "IPAddress": "10.10.10.10",
         "ResourceType": "Login",
-        "resourceName": "test@example.com",
-        "action": "'test@example.com'(with role 'System Admin':'System Admin') logged in via SSO-SAML.",
+        "resourceName": "sanitized_email.com",
+        "action": "'sanitized_email.com'(with role 'System Admin':'System Admin') logged in via SSO-SAML.",
```
Comment (v-rucdu) [inline Sample Data/Custom/PaloAltoPrismaCloudAudit_CL.json:8]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/ProofPointTAPClicksBlocked_CL_sample_data.json [hunk 8]
```diff
@@ -8,10 +8,10 @@
     "Computer": "",
     "RawData": "",
     "clickTime_t": "2020-04-22T20:15:06Z",
-    "sender_s": "sanitized@sanitized.com",
-    "recipient_s": "sanitized@sanitized.com",
+    "sender_s": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/ProofPointTAPClicksBlocked_CL_sample_data.json:11]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/ProofPointTAPClicksBlocked_CL_sample_data.json [hunk 9]
```diff
@@ -8,10 +8,10 @@
...
+    "recipient_s": "sanitized_email.com",
     "senderIP_s": "00.00.00.00",
-    "messageID_s": "<450D4F1E-E216-496A-9D0D-9180EF992779sanitized@sanitized.com>",
+    "messageID_s": "<450D4F1E-E216-496A-9D0D-9180EF992779sanitized_email.com>",
```
Comment (v-rucdu) [inline Sample Data/Custom/ProofPointTAPClicksBlocked_CL_sample_data.json:14]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/ProofPointTAPClicksPermitted_CL_sample_data.json [hunk 10]
```diff
@@ -8,8 +8,8 @@
     "Computer": "",
     "RawData": "",
     "clickTime_t": "2020-04-27T19:50:50Z",
-    "sender_s": "sanitized@sanitized.com",
-    "recipient_s": "sanitized@sanitized.com",
+    "sender_s": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/ProofPointTAPClicksPermitted_CL_sample_data.json:11]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/ProofPointTAPMessagesBlocked_CL_sample_data.json [hunk 11]
```diff
@@ -13,18 +13,18 @@
     "subject_s": "BROOKS (17124244096) left you a message 21 second(s) long.",
     "quarantineRule_s": "module.sandbox.rule.threat",
     "replyToAddress_s": "[]",
-    "toAddresses_s": "sanitized@sanitized.com",
+    "toAddresses_s": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/ProofPointTAPMessagesBlocked_CL_sample_data.json:16]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/ProofPointTAPMessagesDelivered_CL_sample_data.json [hunk 12]
```diff
@@ -13,26 +13,26 @@
     "subject_s": "Your Reporting Statements Are Ready at WCM Investment Management's Client Portal",
     "quarantineRule_s": "module.access.rule.capture_prerewrite",
     "replyToAddress_s": "[]",
-    "toAddresses_s": "[\r\n  \"aw108@abc.com\"\r\n]",
+    "toAddresses_s": "[\r\n  \"sanitized_email.com\"\r\n]",
     "QID_s": "03I5YVvs004766",
-    "sender_s": "sanatized@sanatized.com",
-    "recipient_s": "[\r\n  \"aw108@abc.com\"\r\n]",
+    "sender_s": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/ProofPointTAPMessagesDelivered_CL_sample_data.json:18]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/SenservaPro_CL.json [hunk 13]
```diff
@@ -1,7 +1,7 @@
 [
 	{
 		"ControlName": "UserTest",
-		"Value": "[{\"Tag\":\"UserName\",\"Value\":\"TestUser (test@test.com)\"},{\"Tag\":\"Roles\",\"Value\":{\"$values\":[]}}]",
+		"Value": "[{\"Tag\":\"UserName\",\"Value\":\"TestUser (sanitized_email.com)\"},{\"Tag\":\"Roles\",\"Value\":{\"$values\":[]}}]",
```
Comment (v-rucdu) [inline Sample Data/Custom/SenservaPro_CL.json:4]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/SlackAudit_CL.json [hunk 14]
```diff
@@ -8,7 +8,7 @@
           "user":{
              "id":"U012KR7ESM7",
              "name":"User1",
-             "email":"user@org.com",
+             "email":"sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/SlackAudit_CL.json:11]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/Workplace_Facebook_CL.json [hunk 15]
```diff
@@ -11,9 +11,9 @@
                   "actor_community_id":"504942443827307",
                   "actor_scim_company_id":"504942447160640",
                   "actor_id":"100064094363063",
-                  "actor_email":"user@company.com",
+                  "actor_email":"sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/Workplace_Facebook_CL.json:14]: No need to sanitize this as well, its already dummy email

File: Sample Data/Custom/Zoom_CL.json [hunk 16]
```diff
@@ -10,7 +10,7 @@
    },
    {
       "id":"6OpeTW3TTFKt-JdyNv6j4A",
-      "email":"user@company.com",
+      "email":"sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Custom/Zoom_CL.json:13]: No need to sanitize this as well, its already dummy email

File: Sample Data/Sample Data/Custom/Agari/agari_apdtc_log_CL.json [hunk 17]
```diff
@@ -9,8 +9,8 @@
     "RawData": "",
     "id_g": "52289fcc-28ea-11eb-8461-0242ac140004",
     "timestamp_ms_s": 1605627243000,
-    "to_s": "user1@mydomain.com",
-    "from_s": "User A <usera@domaina.com>",
+    "to_s": "sanitized_email.com",
```
Comment (v-rucdu) [inline Sample Data/Sample Data/Custom/Agari/agari_apdtc_log_CL.json:12]: No need to sanitize this as well, its already dummy email

File: Sample Data/Syslog/CylancePROTECT.json [hunk 18]
```diff
@@ -13104,7 +13104,7 @@
     "Signed": ""
   },
   {
-    "SyslogMessage": "Nov 30 19:04:24 sysloghost CylancePROTECT Event Type: AuditLog, Event Name: LoginSuccess, Message: Provider: CylancePROTECT, Source IP: 10.100.237.101, User: Test User01 (testuser01@abc.com)",
+    "SyslogMessage": "Nov 30 19:04:24 sysloghost CylancePROTECT Event Type: AuditLog, Event Name: LoginSuccess, Message: Provider: CylancePROTECT, Source IP: 10.100.237.101, User: Test User01 (sanitized_email.com)",
```
Comment (v-rucdu) [inline Sample Data/Syslog/CylancePROTECT.json:13107]: No need to sanitize this as well, its already dummy email

File: Sample Data/Syslog/DigitalGuardianDLPEvent.json [hunk 19]
```diff
@@ -8,7 +8,7 @@
...
+        "SyslogMessage": "08:06.380Z ABC-Manager DLP - INCADD incident_id=\"1.12815.10\" managed_device_id=\"1\" number_of_incidents=\"2\" incident_status=\"New,Audit Only\" matched_policies_by_severity=\"High:C_PHI_MRN / C_MRN_>25 \" action_taken=\"NET_NS_H\" matches=\"55\" protocol=\"SMTP\" http_url=\"\" inspected_document=\"Report_3.16-4.17__UPDATED.XLSX\" source=\"sanitized_email.com\" source_ip=\"1.1.1.1\" source_port=\"21752\" destination=\"sanitized_email.com\" destination_ip=\"2.2.2.2\" destination_port=\"25\" email_subject=\"RE: Open Encounters\" email_sender=\"sanitized_email.com\" email_recipients=\"sanitized_email.com \" timestamp=\"2021-06-23 12:06:09 PDT\" incidents_url=https://aaa.lpch.net/LoadIncidentManagement.do?m=1&id=1,27372",
```
Comment (v-rucdu) [inline Sample Data/Syslog/DigitalGuardianDLPEvent.json:11]: No need to sanitize this as well, its already dummy email

**Review by v-rucdu — CHANGES_REQUESTED** [review]

**Comment by preetikr** [comment]
@v-rucdu / @NikTripathi - Is it possible to please do the following for these changes so that the deleted emails do not appear in history: Thanks. 
1.	Copy the impacted sample data file from master branch
2.	Delete the instance (#1) from GitHub
3.	Make changes to #1 by replacing email IDs and anything else you observe that seems like a name or contact info (replace by 1111 or so) etc. 
4.	Then check in #3 as a new file
5.	This way the changes won’t show up and also name the PRs as sample file delete and sample file create.

File: Sample Data/CEF/SymantecDLP.json [hunk 20]
```diff
@@ -5957,7 +5957,7 @@
...
+    "AdditionalExtensions": "BLOCKED=None;APPLICATION_NAME=Microsoft Outlook;INCIDENT_ID=1259;INCIDENT_SNAPSHOT=;https://SAMPLEHOST001.abc.abccompany.com/ProtectManager/EndpointIncidentDetail.do?value(variable_1)=;incident.id&value(operator_1)=;incident.id_in&value(operand_1)=1259;ATTACHMENT_FILENAME=DPA12345.docx ;FILE_NAME=N/A;MATCH_COUNT=2;PROTOCOL=Endpoint Email/SMTP;RECIPIENTS=sanitized_email.com;SENDER=DLP.sanitized_email.com;SUBJECT=FW: test email;SEVERITY=1:High;OCCURRED_ON=October 23, 2020 2:31:58 PM;URL=N/A;DESTINATION_IP=null ;ENDPOINT_MACHINE=L1020088;ENDPOINT_DEVICE_ID=N/A;MACHINE_IP=10.222.248.54;SCAN=N/A;QUARANTINE_PATH=N/A",
```
[the remark this answers is not in the document]
Comment (v-rucdu) [reply Sample Data/CEF/SymantecDLP.json]: As we want to add a validation check for email ids provided in sample data in future, we need to have a specific sanitized email address and hence we are changing for all.

**Review by v-rucdu — APPROVED** [review]

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**